### PR TITLE
Added Usual Stop command and fixed stop command when Overrun Error flag. 

### PR DESCRIPTION
**Pull request by Syunsuke-Okamoto**
# The cpsaio driver updated.
* Added Usual Stop command.
* Fixed stop command when Overrun Error flag.

### CODE REVIEW

File: cps-drivers/aio/cpsaio.c [hunk 1]
```diff
@@ -778,6 +796,24 @@ unsigned long cpsaio_get_status( unsigned char inout, unsigned long BaseAddr, un
...
 		else ulTmpStatus &= ~(CPS_AIO_AIS_SCERR);
 
+
+		// CPS-AI-1608LI / CPS-AI-1608ALI do not stop the sampling analog data
+		// when the overflow error flag or the oversampling error flag is on.
+		// 
+		if( ulTmpStatus & (CPS_AIO_AIS_SCERR | CPS_AIO_AIS_OFERR) )
+		{
+			if( ulTmpStatus & CPS_AIO_AIS_BUSY ){
+				wAnalogStatus = 0;
```
Comment (ura1231) [inline cps-drivers/aio/cpsaio.c:806]: (cpsaio.h)
///< ステータスマクロ
#define CPS_AIO_AIS_BUSY		0x0001
#define CPS_AIO_AIS_START_TRG	0x0002
#define CPS_AIO_AIS_DATA_NUM	0x0010
#define CPS_AIO_AIS_OFERR		0x00010000
#define CPS_AIO_AIS_SCERR		0x00020000

803  if( ulTmpStatus & (CPS_AIO_AIS_SCERR | CPS_AIO_AIS_OFERR) )
804  {
805      if( ulTmpStatus & CPS_AIO_AIS_BUSY ){

803行目でフィルタされるため、805行目の条件が真になることはありません。
意図通りの実装であるか、確認をお願いします。

**Review by ura1231 — CHANGES_REQUESTED** [review]
１点確認事項を追記しました。
確認をお願いします。

File: cps-drivers/aio/cpsaio.c (hunk 1 above)
Comment (Syunsuke-Okamoto) [reply cps-drivers/aio/cpsaio.c:806]: Thanks you your comment.
I checked these codes.
These code is running right.

At first, line 803 means  : 

| pattern [hex] | true / false |
|:---|:---|
| 10000 | true |
| 20000 | true |
| 10001 | true |
| 20001 | true |
| 00001 | false |

This does not pass  AIS_BUSY only.
This need pass OF_ERR or SC_ERR.

At, second line 805 means : 

| pattern [hex] | true / false |
|:---|:---|
| 10000 | false |
| 20000 | false |
| 10001 | true |
| 20001 | true |

This need pass (OF_ERR and AIS_BUSY) or (SC_ERR and AIS_BUSY).

Best regards,

**Review by ura1231 — APPROVED** [review]

File: cps-drivers/aio/cpsaio.c (hunk 1 above)
Comment (ura1231) [reply cps-drivers/aio/cpsaio.c:806]: I confirmed to be your intended design.

**Pull request merged**